service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -86,14 +91,20 @@ void aboutToSubmitShouldPopulateHiddenHearingDateField() throws Exception {
                     .label(date.plusDays(5).toString())
                     .build());
 
-        Map<String, Object> data = ImmutableMap.of("cmoHearingDateList", dynamicHearingDates);
+        Map<String, Object> data = ImmutableMap.of("cmoHearingDateList", dynamicHearingDates,
+            "allPartiesCustom", createUnassignedDirection());
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = getResponse(data, "about-to-submit");
 
         assertThat(callbackResponse.getData()).doesNotContainKey("cmoHearingDateList");
 
         CaseData caseData = mapper.convertValue(callbackResponse.getData(), CaseData.class);
+        CaseManagementOrder caseManagementOrder = caseData.getCaseManagementOrder();
 
+        List<Element<Direction>> responseDirections = caseManagementOrder.getDirections();
+
+        assertThat(responseDirections.size()).isEqualTo(1);
+        assertThat(responseDirections).isEqualTo(createCustomDirection(ALL_PARTIES));
```

Comment:
I would just say asserThat(caseData.getCaseManagementOrder().getDirections() here. No point extracting to all these variables